pkgs/top-level/python-packages.nix

```
@@ -5054,6 +5062,8 @@ in {
 
   minidump = callPackage ../development/python-modules/minidump { };
 
+  miniKanren = callPackage ../development/python-modules/miniKanren { };
```

```suggestion
  minikanren = callPackage ../development/python-modules/minikanren { };
```
also please update pname.

---

Is it wrong to keep it in camel case ? The package on Pypi is called [`miniKanren`](https://pypi.org/project/miniKanren/), so I would think it's best to keep it the same in nixpkgs, if possible.

---

https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#contributing-guidelines-contributing-guidelines

> Attribute names in python-packages.nix as well as pnames should match the library's name on PyPI, but be normalized according to [PEP 0503](https://www.python.org/dev/peps/pep-0503/#normalized-names). This means that characters should be converted to lowercase and . and _ should be replaced by a single - (foo-bar-baz instead of Foo__Bar.baz). If necessary, pname has to be given a different value within fetchPypi.